src/components/Accordion/Accordion/__tests__/accordion.interactions.js

```diff
@@ -0,0 +1,54 @@
+import { expect } from "@storybook/jest";
+import { userEvent, within } from "@storybook/testing-library";
+import { delay, interactionSuite, resetFocus } from "../../../../__tests__/interactions-helper";
+
+const CHANGES_DELAY = 1;
+
+function getAccordionHeadingBtText(canvas, title) {
+  return canvas.getByText(title)?.closest("button");
+}
+
+function getOpenedAccordionItem(canvas) {
+  const elPanel = canvas.getByRole("region");
+  const elHeading = within(elPanel.parentNode).getByRole("button");
+  return { elPanel, elHeading };
+}
+
+const openCloseAccordionSingleActiveTests = async canvas => {
+  // prepare: take sizes of slider and waiting for render
+  let elHeading, elPanel;
+  await delay(CHANGES_DELAY);
+
+  // try to click on already selected Accordion Item heading
+  ({ elHeading, elPanel } = getOpenedAccordionItem(canvas));
+  const before = elPanel.id;
+  userEvent.click(elHeading);
+  ({ elHeading, elPanel } = getOpenedAccordionItem(canvas));
+  const after = elPanel.id;
+  // what was opened should be still opened
+  await expect(before).toBe(after);
+  // panel and heading aria controls are the same
+  await expect(elHeading.getAttribute("aria-controls")).toBe(elPanel.id);
+
+  // select first (0) AccordionItem
+  elHeading = getAccordionHeadingBtText(canvas, "Notifications");
+  userEvent.click(elHeading);
+  elPanel = canvas.getByRole("region");
+  await expect(elHeading.getAttribute("aria-expanded")).toBe("true");
+  await expect(elHeading.getAttribute("aria-controls")).toBe(elPanel.id);
+  await delay(CHANGES_DELAY);
+
+  // select back second (1) AccordionItem
+  elHeading = getAccordionHeadingBtText(canvas, "Setting");
+  userEvent.click(elHeading);
+  elPanel = canvas.getByRole("region");
+  await expect(elHeading.getAttribute("aria-expanded")).toBe("true");
+  await expect(elHeading.getAttribute("aria-controls")).toBe(elPanel.id);
```

Comment:
would you like to check here as well that `aria-expanded` of the first item is false?

Reply:
added:
https://github.com/mondaycom/monday-ui-react-core/pull/617/commits/86990168c387433e2dd28b902c12bba8cc86485d